src/components/Filterbar/FilterBar.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react'
+import { IoReload } from "react-icons/io5";
+import { FaRegStar } from "react-icons/fa";
+import "../Filterbar/FilterBar.css";
+
+ export const FilterBar = () => {
+  return (
+    <form className="sidebar-container sidebar-content">
+      <ul className="sidebar-items">
+         {/* <!-- ----------sort--------------------------- --> */} 
+        <span className="filter-icon">
+          <IoReload type='reset'/>
+        </span>
+        <h4 className="sidebar-content-item">Sort</h4>
+        <li className="sidebar-list">
+          <input
+            type="radio"
+            value="price low to high"
+            name="sort-btn"            
+          />
+          Price Low to High
+        </li>
+        <li className="sidebar-list">
+          <input
+            type="radio"
+            value="price high to low"
+            name="sort-btn"            
+          />
+          Price High to Low
+        </li>
+        {/* -----------price range------- */}
+        <h4 className="sidebar-content-item">PRICE RANGE</h4>
+        <div className="sidebar-list">
+          <li>
+            <input
+              type="range"           
+              max={15000}
+              min={50}
+              steps={1000}
+            />
+          </li>
+        </div>
+        {/* <!-- -------------------CATEGORY-------------- --> */}
+        <h4 className="sidebar-content-item">CATEGORY</h4>
+        <div className="sidebar-list">
+          <li>
+            <input
+              type="radio"
+              value="Men"
+              name="category-btn"              
+            />
+            Men
+          </li>
+          <li className="sidebar-list">
+            <input
+              type="radio"
+              value="Women"
+              name="category-btn"              
+            />
+            Women
+          </li>
+        </div>
+
+        {/* <!-- -------------------Availability-------------- --> */}
+        <h4 className="sidebar-content-item">Availability</h4>
+        <li className="sidebar-list">
+          <input
+            type="checkbox"
+            value="In-stock"
+            name="available-btn"            
+          />
+          Include Out-of-Stock
+        </li>
+        <li className="sidebar-list">
+          <input
+            type="checkbox"
+            value="out-ot-stock"
+            name="available-btn"            
+          />
+          Fast Delivery
+        </li>
+
+        {/* <!-- -------------------Brand-------------- --> */}
+        <h4 className="sidebar-content-item">Brand</h4>
+        <li className="sidebar-list">
```

Comment:
You can add all the brands in a array 
```const brands = ["brand name1", " brand name 2"]```

then you can map through this list and return an element for echa brand.

```{brands.map(brand=> <li>{}</Li>)}```
In this way you can avoid duplicate html code.

Reply:
will try it